libs/template/themes/icons/mappers.ts

```diff
@@ -0,0 +1,26 @@
+import { IconMapper } from '@spryker-oryx/experience';
+import { IconTypes } from './icon.model';
+
+export const materialMapper: IconMapper = {
```

Comment:
Can we move the mapper to the presets/resources/icons?

Reply:
Unfortunately no. In this case we have circular dependency